### DISCUSSION TITLE
LoRA训练时Loss一直是0

### DISCUSSION THREAD

**Discussion by sofiane-20241050** [zai-org/ChatGLM3#921] [Q&A]
请问为什么我在使用LoRA训练时，Loss从第二个step开始一直是0.0

**Comment by Wenting1227** [comment]
我之前也遇到过，是peft库的版本问题，peft库的版本得是0.7.1

**Comment by sofiane-20241050** [comment]
ok，我也是重新指定PEFT库的版本后，训练时loss正常了

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年03月07日 16:38 |
| 收件人 | THUDM/ChatGLM3 ***@***.***> |
| 抄送人 | sofiane-20241050 ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [THUDM/ChatGLM3] LoRA训练时Loss一直是0 (Discussion #921) |

我之前也遇到过，是peft库的版本问题，peft库的版本得是0.7.1
